buildSrc/src/main/kotlin/GitInfo.kt

```diff
@@ -0,0 +1,14 @@
+import org.gradle.api.Project
+import java.io.ByteArrayOutputStream
+
+fun Project.runCommand(command: String): String {
+    val byteOut = ByteArrayOutputStream()
+    project.exec {
+        commandLine = listOf( "sh", "-c", command)
+        standardOutput = byteOut
+    }
+    return byteOut.toString()
+}
+
+fun Project.gitLatestTag(): String =
+        runCommand("git describe --tags --abbrev=0").trim()
```

Comment:
nitpick: use 4-space indentation

```suggestion
    runCommand("git describe --tags --abbrev=0").trim()
```